EasyImgur/ImgurAPI.cs

```diff
@@ -304,9 +305,18 @@ static public APIResponses.AlbumResponse UploadAlbum(Image[] _Images, string _Al
 
             // in case I need them later 
             List<APIResponses.ImageResponse> responses = new List<APIResponses.ImageResponse>();
-            for (int i = 0; i < _Images.Count(); ++i)
+            for (int i = 0; i < _Paths.Count(); ++i)
             {
-                Image image = _Images[i];
+                Image image = null;
+                try
+                {
+                    image = Image.FromStream(new System.IO.MemoryStream(System.IO.File.ReadAllBytes(_Paths[i])));
```

Comment:
Can `Image.FromStream` return `null`?

Reply:
My mistake, the code is suppose to check whether `albumCoverImg` has been assigned or not.  This will be fixed.

Reply:
Alright, thanks for clarifying! I reckon it's best to just push a new commit to this branch.